### PR TITLE
DT/cameracontrols: a few fixes

### PR DESCRIPTION
**Pull request by CST1229**
This makes the following changes to Camera Controls:
- Fix touching mouse-pointer blocks (#349)
- Remove speech bubble bounds (#743, #741)
- Reset camera when the project is loaded
- Change a bunch of `let`s to `const`s in the code

Debug project (sb3 renamed to zip): [Camera test.zip](https://github.com/TurboWarp/extensions/files/12446932/Camera.test.zip)

### CODE REVIEW

File: extensions/DT/cameracontrols.js [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
   vm.runtime.runtimeOptions.fencing = false;
   vm.renderer.offscreenTouching = true;
-  vm.setInterpolation(false);
+  vm.setInterpolation(0);
```
Comment (GarboMuffin) [inline extensions/DT/cameracontrols.js:30]: dont think that's intentional?

Comment (LilyMakesThings) [reply extensions/DT/cameracontrols.js:30]: It is, "setInterpolation" apparently accepts number values, not booleans

Comment (CST1229) [reply extensions/DT/cameracontrols.js:30]: The types say so:
image

Comment (GarboMuffin) [reply extensions/DT/cameracontrols.js:30]: the types are wrong https://github.com/TurboWarp/extensions/pull/961

File: extensions/DT/cameracontrols.js [hunk 2]
```diff
@@ -61,20 +61,133 @@
...
+  function _translateY(y, fromTopLeft = false, multiplier = 1, doZoom = true) {
+    const h = fromTopLeft ? (vm.runtime.stageHeight / 2) : 0;
+    return ((y - h) / (doZoom ? (cameraZoom / 100) : 1)) + h;
+  }
+
+  function translateX(
+    x, fromTopLeft = false, multiplier = 1, doZoom = true,
+    y = 0, yMult = multiplier
+  ) {
+    // rotation hell, not gonna bother with it for now
```
Comment (GarboMuffin) [inline extensions/DT/cameracontrols.js:100]: you've got some time to bother with it if you want to

**Comment by GarboMuffin** [comment]
done?

**Comment by CST1229** [comment]
> done?

i'd say it's ready to be merged. eventually i want to fix pen too, but that can wait for a future PR.

File: extensions/DT/cameracontrols.js [hunk 3]
```diff
@@ -240,6 +395,17 @@
             blockType: Scratch.BlockType.REPORTER,
             text: "camera direction",
           },
+          // debugging blocks
+          {
```
Comment (GarboMuffin) [inline extensions/DT/cameracontrols.js:399]: supposed to be commented?

**Pull request merged**